cloud/pkg/cloudhub/channelq/channelq.go

```diff
@@ -316,9 +316,9 @@ func (q *ChannelMessageQueue) Publish(msg *beehiveModel.Message) error {
 	case application.MetaServerSource:
 		beehiveContext.Send(modules.DynamicControllerModuleName, *msg)
 	case model.ResTwin:
-		beehiveContext.SendToGroup(model.SrcDeviceController, *msg)
+		beehiveContext.SendToGroup(modules.DeviceControllerModuleGroup, *msg)
 	default:
-		beehiveContext.SendToGroup(model.SrcEdgeController, *msg)
+		beehiveContext.SendToGroup(modules.EdgeControllerGroupName, *msg)
```

Comment:
Is it possible to keep having the same naming format.
e.g.
`DeviceControllerModuleGroup` and `EdgeControllerModuleGroup`
e.g.
`DeviceControllerGroupName` and `EdgeControllerGroupName`

Reply:
Thanks a lot, it will be better if you have time to fix it.